### ISSUE TITLE
Tested 0.4.0 pre-release APP2-C 15.6RC

### ISSUE THREAD

**Issue by Dravegon** [LNSSPsd/PodsGrant#74]
Hello, i tried the latest 0.4.0 pre-release on Palera1n rootful 15.6RC on USB-C version of AirPods Pro 2 and it only shows the pop-up screen.

Can i have full functionality on USB-C version of AirPods Pro 2 on Palera1n rootful 15.6RC when i share my bluetoothd file?

Thanks.

**Comment by GovenorKratos** [comment]
You won’t get the full functionality of your AirPods Pro 2 as these came out with iOS 16. However, you can have them treated as Pro 1 and get some functionality at least.

**Comment by Dravegon** [comment]
> You won’t get the full functionality of your AirPods Pro 2 as these came out with iOS 16. However, you can have them treated as Pro 1 and get some functionality at least.

Yeah thanks, i have it already. Btw what's weird on my newer iPhone with iOS 16 on it and Dopamine the Airpods didn't work natively so i had to use this PodsGrant tweak as well,weird right?

**Comment by GovenorKratos** [comment]
Not really, there’s a distinct difference between the AirPods Pro 2 USB C and the original ones. The USB C ones only work on iOS 17+

**Comment by Dravegon** [comment]
> Not really, there’s a distinct difference between the AirPods Pro 2 USB C and the original ones. The USB C ones only work on iOS 17+

Oh i see, yeah now you saying before i had my older iPhone X on iOS 16 and then it worked natively but that was the older AirPods Pro 2 non USB C version Thanks🙂
